src/pages/transferWritingPricePage/enterAccountInfo/EnterAccountInfo.tsx

```diff
@@ -0,0 +1,156 @@
+import { useEffect, useRef, useState } from "react";
+import * as S from "./EnterAccountInfo.style";
+import { BANK_LIST } from "@/constants/bank";
+import { AnimatePresence, useAnimation } from "framer-motion";
+import { useLocation } from "react-router-dom";
+import { PATH } from "@/constants/path";
+import useToastConfig from "@hooks/common/useToastConfig";
+import usePreventLeave from "@hooks/common/usePreventLeave";
+
+const EnterAccountInfo = ({
+  bank,
+  accountNumber,
+  onSubmitAccount,
+  onSetAccountNumber,
+  onSetBank,
+}: {
+  bank: string | null;
+  accountNumber: string | null;
+  onSubmitAccount: React.Dispatch<React.SetStateAction<"none" | "enter">>;
+  onSetAccountNumber: React.Dispatch<React.SetStateAction<string | null>>;
+  onSetBank: React.Dispatch<React.SetStateAction<string | null>>;
+}) => {
```

Comment:
nullable한 타입은 헬퍼타입으로 만들어서 사용할 수도 있을 것 같네용~!
지난 플젝에서 피드백 받은 부분인데

```typescript
// type/nullable.ts
export type Nullable<T> = T | null
``` 

이렇게 헬퍼타입을 만들어서 이렇게 사용할 수 있을 것 같습니다

```suggestion
  bank: Nullable<string>;
  accountNumber:  Nullable<string>;
  ....
```

Reply:
좋은 방법인거 같아요! 이 부분 반영하겠습니다!!